### PR TITLE
allow to disable apps building

### PR DESCRIPTION
**Pull request by vladak**
This trivial change allows to disable apps building.  It's handy for minimization.

The option was already there and adhered to by `apps/build.info`, however Configure did not allow to use it directly, only via cascading.

### CODE REVIEW

**Comment by mattcaswell** [comment]
Any new option should be documented in INSTALL.md

**Comment by vladak** [comment]
> Any new option should be documented in INSTALL.md

Fixed.

File: INSTALL.md [hunk 1]
```diff
@@ -583,6 +583,10 @@ access to algorithm internals that are not normally accessible.
 Additional information related to ACVP can be found at
 <https://github.com/usnistgov/ACVP>.
 
+### no-apps
+
+Do not build apps, e.g. the openssl program. This is handy for minimization.
```
Comment (mattcaswell) [inline INSTALL.md:588]: Probably we should make it clear that this option also disables the tests - so "make test" will no longer work.

**Comment by t8m** [comment]
We would need a run-checker target for that no-apps build option and I assume you'll find that many tests will fail without apps so you'll need to fix those failures, which will be a quite tedious patch to do.

**Comment by mattcaswell** [comment]
>  I assume you'll find that many tests will fail without apps so you'll need to fix those failures, which will be a quite tedious patch to do.

Nope. "no-apps" cascades to "no-tests" so the tests simply don't run at all.

**Comment by t8m** [comment]
> Nope. "no-apps" cascades to "no-tests" so the tests simply don't run at all.

Ah, right. That makes the option pretty limiting.

File: INSTALL.md (hunk 1 above)
Comment (vladak) [reply INSTALL.md:588]: Ok, note added.

**Comment by mattcaswell** [comment]
> We would need a run-checker target for that no-apps build option

@vladak - as per @t8m's feedback please could you add an entry for this in .github/workflows/run-checker-daily.yml

**Comment by vladak** [comment]
> > We would need a run-checker target for that no-apps build option
> 
> @vladak - as per @t8m's feedback please could you add an entry for this in .github/workflows/run-checker-daily.yml

Sure, added.

**Review by mattcaswell — APPROVED** [review]

**Review by tom-cosgrove-arm — APPROVED** [review]
LGTM

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Squashed and merged to master branch. Thank you for your contribution.

**Pull request closed without merging**